src/style.css

```diff
@@ -1 +1,67 @@
 /* Styles go here */
+* {
+  margin: 0;
+  box-sizing: border-box;
+  padding: 0;
+}
+
+html {
+  max-width: 1200px;
+  min-width: 1024px;
+}
+
+.head {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 60px;
```

Comment:
better to not hardcode height
better to give the possibility to stretch it by child elements

Reply:
ok, thanks
i will correct